loc/US/nomads_strings_core.lua

```diff
@@ -68,15 +68,15 @@ tooltip_lobby_unit_restriction_nonomads_desc = ""
 tooltip_radar_boost_title = "Radar Boost Toggle"
 tooltip_radar_boost_desc = "Starts/stops charging radar boosting for the selection units. When finished charging up radar boost initiates automatically. Charging up requires energy."
 tooltip_bombardmode_title = "Bombard Mode Toggle"
-tooltip_bombardmode_desc = "Turn the selection units bombard mode on/off"
+tooltip_bombardmode_desc = "bombard mode decrese reload time by 50% but increse firing randomnes"
```

Comment:
plz fix spelling errors:
"Bombardment mode decreases the reload time by 50% but increases the firing randomnes."